bin/archive-meeting

```diff
@@ -335,11 +335,9 @@ def find_latest_weekly_notes(target_dir:)
 end
 
 def select_meeting_notes_section(latest_weekly_notes:, summary_file:, llm_model:)
-  weekly_notes_content = File.read(latest_weekly_notes)
-  executive_summary_content = File.read(summary_file)
-  LlmWorkflowUtils.select_meeting_notes_section(
-    weekly_notes_content: weekly_notes_content,
-    executive_summary_content: executive_summary_content,
+  MeetingFileUtils.select_meeting_notes_section(
```

Comment:
I don't think MeetingFileUtils.select_meeting_notes_section should even be an extracted method, the logic should just be here and then it can be removed from MeetingFileUtils.